src/core/mount.c

```diff
@@ -747,7 +747,8 @@ static int mount_spawn(Mount *m, ExecCommand *c, pid_t *_pid) {
                 return r;
 
         exec_params.environment = UNIT(m)->manager->environment;
-        exec_params.flags |= UNIT(m)->manager->confirm_spawn ? EXEC_CONFIRM_SPAWN : 0;
+        exec_params.confirm_spawn = manager_get_confirm_spawn(UNIT(m)->manager);
+        exec_params.flags |= exec_params.confirm_spawn ? EXEC_CONFIRM_SPAWN : 0;
```

Comment:
This means that we carry the same information twice: once as the parameter, and once as the boolean bit. It's better to avoid that. Just drop the EXEC_CONFIRM_FLAG, we don't need it. Users of that flags should just check `exec_params.confirm_spawn`.